content/management/configuration-api/v3.2/index.mdx

```diff
@@ -33,31 +32,338 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
 
 All configurations set by this API will have action in system after max 2 minutes. This delay will be removed in the future.
 
+# Groups
+
+Groups let you organize your work by creating teams with Agent and Bot members. You can use groups to do basic configuration and separate chat routing. It worth mentioning that one Agent can belong to mutliple groups at the same time. It's possible to configure routing priorities for Agents and Bots specifically per group.
```

Comment:
We can still modify this description if you like.
I've changed _common configuration_ to _basic configuration_ because that's how I interpreted the meaning of it. Correct me if you meant differently :)

Reply:
When I used "common", I meant grouped/shared settings - agents A, B and C speak English and work 8-16 so we join them into a group. WDYT?

Reply:
Besides that, I like your changes in description :) very readable